Wikipedia/Code/WMFAccountCreationViewController.swift

```diff
@@ -97,8 +101,34 @@ class WMFAccountCreationViewController: UIViewController, WMFCaptchaViewControll
         
         loginButton.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(self.loginButtonPushed(_:))))
         titleLabel.text = localizedStringForKeyFallingBackOnEnglish("navbar-title-mode-create-account")
+       
+        captchaTitleLabel.text = localizedStringForKeyFallingBackOnEnglish("account-creation-captcha-title")
+        
+        // Reminder: used a label instead of a button for subtitle because of multi-line string issues with UIButton.
+        captchaSubtitleLabel.attributedText = getCaptchaSubtitleAttributedString()
+        captchaSubtitleLabel.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(self.requestAnAccountTapped(_:))))
+
+        view.wmf_configureSubviewsForDynamicType()
     }
 
+    func requestAnAccountTapped(_ recognizer: UITapGestureRecognizer) {
+        wmf_openExternalUrl(URL.init(string: "https://en.wikipedia.org/wiki/Wikipedia:Request_an_account"))
```

Comment:
Good point. I just double-checked with Carolyn and she said this is ok for now (until we can figure out the other pages).